java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -287,6 +292,9 @@ public static Object[] transactionTestResult() {
             3L, // pfcount(new String[] { hllKey1, hllKey2 });;
             OK, // pfmerge(hllKey3, new String[] {hllKey1, hllKey2})
             3L, // pfcount(new String[] { hllKey3 })
+            15L,
```

Comment:
add comments here to understand which commands these results belong to
It is extremely difficult to update or fix without comments

Reply:
Thanks! Addressed in https://github.com/Bit-Quill/glide-for-redis/pull/263/commits/2575b0125be5640b722ee685c33c9fd624ae1b00